src/utils/knownNetworks.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2018-present, Leap DAO (leapdao.org)
+ *
+ * This source code is licensed under the GNU GENERAL PUBLIC LICENSE Version 3
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+export const KNOWN_NETWORKS = {
+  1: {
+    name: 'Mainnet',
+    provider: {
+      http: 'https://mainnet.infura.io/v3/f039330d8fb747e48a7ce98f51400d65',
+      ws: 'wss://mainnet.infura.io/ws/v3/f039330d8fb747e48a7ce98f51400d65',
+    },
+    etherscanBase: 'https://etherscan.io',
+  },
+  3: {
+    name: 'Ropsten',
+    provider: {
+      http: 'https://ropsten.infura.io',
+      ws: 'wss://ropsten.infura.io/ws',
+    },
+    etherscanBase: 'https://ropsten.etherscan.io',
+  },
+  4: {
+    name: 'Rinkeby',
```

Comment:
Rinkeby and the other ones don't need an apiKey?

Reply:
yeah, good point